hail/python/hailtop/hailctl/config/cli.py

```diff
@@ -96,10 +116,13 @@ def unset(parameter: str):
         del config[section][key]
         with open(config_file, 'w', encoding='utf-8') as f:
             config.write(f)
+    else:
+        print(f"Error: unknown parameter {parameter!r}", file=sys.stderr)
+        sys.exit(1)
```

Comment:
Ah, let's just allow this. I don't see a problem with `unset`ing a value that is not currently set.